express-post-json/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+
+var nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+
+  gradeArray.push(grades);
```

Comment:
Here you should be looping through the `grades` object using a `for in` loop to push each grade into `gradeArray`

Reply:
did you mean like this?